### ISSUE TITLE
Certain filenames are not handled correctly

### ISSUE THREAD

**Issue by iancourt** [XyLe-GBP/mca-coh-gui#2]
It appears some filenames are not handled correctly, notably those with extensions, other than _boot.bin_ and _title.txt_ .

For example, when attempting to dump files from a Tekken 5: Dark Resurrection card, these filenames get truncated:

- IOPRP302.246 --> IOPRP302.
- IRXPK246.302 --> IRXPK246.
- T55DATA0.BIN -->T55DATA
- T55GAME.KEY --> T55GAME (this creates a conflict, as there is also a file named T55GAME with no extension)

Consequently, they fail to dump.

![image](https://github.com/XyLe-GBP/mca-coh-gui/assets/8887641/95f7e301-d242-477f-a40d-f86f7b176202)

For reference, the files on the right of the above screenshot are those dumped from a card image using command line tools.

**Comment by XyLe-GBP** [comment]
Thank you very much for reporting the problem.
I do not have all Sys246/256 dongles, so this was a blind spot.
Is this a glitch that only occurs with the Tekken dongle?
Please provide us with additional details so we can fix it.

**Comment by iancourt** [comment]
Of course, please give me some time. I will attempt further tests and report back with my findings.
